broker-daemon/broker-rpc/wallet-service/commit-balance.js

```diff
@@ -71,9 +71,19 @@ async function commitBalance ({ params, relayer, logger, engines }, { EmptyRespo
     throw new PublicError(`Maxium balance of ${MAX_CHANNEL_BALANCE} exceeded for committing to the relayer. Please try again.`)
   }
 
+  const {maxBalance: maxOutboundBalance} = await engine.getMaxChannel()
+  const {maxBalance: maxInboundBalance} = await inverseEngine.getMaxChannel({outbound: false})
+  const convertedBalance = convertBalance(balance, symbol, inverseSymbol)
+  if (maxOutboundBalance || maxInboundBalance) {
+    if ((maxOutboundBalance && Big(maxOutboundBalance).lte(balance)) || (maxInboundBalance && Big(maxInboundBalance).lte(convertedBalance))) {
```

Comment:
I would separate this if statement so we can use `debug` logs to provide more information. Example:

```
// If the outbound balance isn't sufficient
if (!maxOutboundBalance) throw new PublicError('no channels exist or something')
if (Big(maxOutboundBalance).lt(balance)) {
  this.logger.debug('The outbound balance was less than the balance specified or something', { maxOutboundBalance, balance })
  throw new PublicError('.........')
}
etc...
```

Reply:
We wouldn't throw an error if the channels did not exist, but I get the gist.